mne/epochs.py

```diff
@@ -879,9 +873,14 @@ def standard_error(self, picks=None):
 
         Parameters
         ----------
-        picks : array-like of int | None
-            If None only MEG, EEG, SEEG, ECoG, and fNIRS channels are kept
-            otherwise the channels indices in picks are kept.
+        picks : str | list | slice | None
+            Channels to include. Slices and lists of integers will be
+            interpreted as channel indices. In lists, channel *type* strings
+            (e.g., ``['meg', 'eeg']``) will pick channels of those
+            types, channel *name* strings (e.g., ``['MEG0111', 'MEG2623']``
+            will pick the given channels. Can also be the string values
+            "all" to pick all channels, or "data" to pick data channels.
+            If None (default), all data channels will be used.
```

Comment:
@jona-sassenhagen @cbrnr @agramfort can you see if you are happy with this docstring? Once we converge on it, I'll replace all of the `picks : XXX` that I've slotted in, then this should be ready for merge (only the last line, about what None means in a given context, should vary in most cases).

In the meantime I'll probably push a few commits to fix any CI errors, but we can iterate on this in the meantime.

Reply:
Also @mmagnuski you had comments earlier, too

Reply:
I like the docstring in general. Just a few questions:
* it seems to suggest that you can't do `picks='meg'` (these strings are discussed only in the context of lists).
* is `'all'` really needed? it does the same as `None` .

Reply:
`'all'` is not the same as `None` -- check the docstring. `'all'` is all channels, `None` is all *data* channels.

Reply:
Right, but then `None` and `"data"` do the same thing. :)

Reply:
Can "data" be a default in this case? If not then I'm ok with it as it is. :)

Reply:
Yes here `None` is the same as `"data"`, in other functions `None` will be the same as `"all"`, etc. For backward compatibility, we need to support `None` behaving this way. We could look into switching all of them to explicit defaults with or after this PR (e.g., use "data" instead of `None`) at some point, which would be nicer.

Reply:
Sure, all clear now. So I'd only consider this one slight clarification with regards to str like `"meg"`, which are currently described only in the context of lists of str.